version/1/4/mk/code_of_conduct.md

```diff
@@ -0,0 +1,47 @@
+# Пакт за Придонесување - Kодекс на Oднесување
```

Comment:
What do you think about using "Договор за Соработник - Kодекс на Oднесување" as title? "Пакт за Придонесување" does not reads right because it's Contributor, not Contribution. Maybe "Пакт за Соработник" if using the word "Пакт" there is important?